keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -29,6 +31,7 @@ import (
 	"strings"
 
 	"github.com/blang/semver"
+	"github.com/schollz/progressbar/v3"
```

Comment:
Where need it? I see too much pkgs imported, please double check them really necessary

Reply:
In order to have a good user experience, add the progress bar.
If the network does not allow
Can't see the progress, it affects the experience

Reply:
Useful logs are enough i think.

Reply:
If the network is too bad, the log is stuck all the time.
And we don't know when it will be ready.
It's a better experience